docs/ddp/README.md

```diff
@@ -29,6 +29,9 @@ For Intel® Ethernet Controller X710, a DDP package can be loaded into the NIC u
 ## Short step by step configuration for E810 & X710 series NICs
 ### Install DDP packages
 #### Intel® Ethernet Controller E810
+ddptool : Version 1.0.0.0 Doesn't support DDP on E810 series NICs.
```

Comment:
I think, it would be better to specify the min version of this software that will work for E810 instead of naming DDPTool which is an internal component and from a user perspective is transparent.

Reply:
Do you mean the min version of SRIOV DP?
if so, it would be the version before #348  was merged.
As it was the old images that contained an old dpptool version.

Reply:
yes, min version of SRIOV DP. The latest tagged release 3.3.2 should be the min version which support E800 series DDP.
>Also, it was the ddptool that was the issue, and why it didn't work. so specifying the version of DP I think is irrelevant.

DDPTool is included in SRIOV DP (see images folder). From a user perspective, they don't care about version of DDPTool  version. We do not want users to mix and match this tool but take the DDPTool version which comes with this software, so from their perspective, they only care about what ver of DP works with E800 series.

Reply:
Okay I understand now, that makes sense.
Since the "fix" was done recently and is not in the 3.3.2 release. Should this doc update be put on hold till next release?

Reply:
Done.